vermouth/tests/test_itp_files.py

```diff
@@ -15,6 +15,7 @@
 Test that force field files are properly read.
 """
 
+from inspect import ismethod
```

Comment:
Unused import

Reply:
ahh shoot yes that was my original idea for the last test. I'll remove it.